SPECS/pyproject-rpm-macros/pyproject-rpm-macros.spec

```diff
@@ -41,25 +53,42 @@ Source303:      test_RECORD
 Source901:      README.md
 Source902:      LICENSE
 
-%if 0%{?with_check}
-BuildRequires:  python3-atomicwrites
-BuildRequires:  python3-attrs
-BuildRequires:  python3-pip
-BuildRequires:  python3-pluggy
-BuildRequires:  python3-six
+%if %{with tests}
+BuildRequires:  python3dist(pytest)
+%if %{with pytest_xdist}
+BuildRequires:  python3dist(pytest-xdist)
+%endif
+BuildRequires:  python3dist(pyyaml)
 BuildRequires:  python3dist(packaging)
 BuildRequires:  python3dist(pip)
-BuildRequires:  python3dist(pyyaml)
 BuildRequires:  python3dist(setuptools)
+%if %{with tox_tests}
 BuildRequires:  python3dist(tox-current-env) >= 0.0.6
+%endif
 BuildRequires:  python3dist(wheel)
+#BuildRequires:  (python3dist(tomli) if python3 < 3.11)
 %endif
 
-Requires:       %{_bindir}/find
-Requires:       /bin/sed
+# We build on top of those: (all 3 of the following provided by "azurelinux-rpm-macros")
+BuildRequires:  python-rpm-macros
+BuildRequires:  python-srpm-macros
+BuildRequires:  python3-rpm-macros
 Requires:       python-rpm-macros
 Requires:       python-srpm-macros
 Requires:       python3-rpm-macros
+# FIXME - our toolkit can't handle logical deps like this
+#Requires:       (pyproject-srpm-macros = %{?epoch:%{epoch}:}%{version}-%{release} if pyproject-srpm-macros)
+Requires:       pyproject-srpm-macros = %{version}-%{release}
+
+# We use the following tools outside of coreutils
+Requires:       findutils
+Requires:       sed
+
+# This package requires the %%generate_buildrequires functionality.
+# It has been introduced in RPM 4.15 (4.14.90 is the alpha of 4.15).
+# What we need is rpmlib(DynamicBuildRequires), but that is impossible to (Build)Require.
+Requires:       rpm-build >= 4.14.90
+BuildRequires:  rpm-build >= 4.14.90
```

Comment:
Not needed: we always had a newer version of RPM. Also, we ignore BR produced by `generate_buildrequires`, so this may be misleading.

Reply:
fixed